app/controllers/webhooks_controller.rb

```diff
@@ -0,0 +1,27 @@
+class WebhooksController < ApplicationController
+  include ShopifyApp::WebhookVerification
+
+  class ShopifyApp::MissingWebhookJobError < StandardError; end
+
+  def receive
+    job_args = {shop_domain: shop_domain, webhook: webhook}
+    webhook_job_klass.perform_later(job_args)
+    head :ok
+  end
+
+  private
+
+  def webhook
+    params.except(:controller, :action, :type)
+  end
+
+  def webhook_job_klass
+    "#{webhook_type.classify}Job".constantize
```

Comment:
> This should be just constantize

Careful with `constantize`, it respects Ruby's constant lookup which is generally not what you want. In this case it _probably_ doesn't change a thing, but it's better to keep sane habits. Also, testing against `nil` isn't very hard, and it's faster than exception-based flow control. 

Ref: http://api.rubyonrails.org/classes/ActiveSupport/Inflector.html#method-i-safe_constantize